tetris.sh

```diff
@@ -69,7 +69,9 @@ while getopts ":l:h" opt; do
   case ${opt} in
     l )
         langChocie=${OPTARG};
-        . lang/$langChocie.sh &> /dev/null
+        if [ "$langChocie" != "english" ]; then
```

Comment:
It is probably time to introduce new variable ```script_dir```:
```
script_dir=$(cd $(dirname $0) && pwd)
```
it should be set somewhere at the top of the script.
I suggest to use
```
[ "$user_lang" != "english" ] && {
    [ -r "$script_dir/lang/$user_lang"] || {
        echo "$script_dir/lang/$user_lang doesn't exist"
        exit 1
    }
    . $script_dir/lang/$user_lang || {
        echo "Failed to load $script_dir/lang/$user_lang"
        exit 1
    }
}
```
Couple of notes here. user_lang is probably a better name for the variable. Please note that all functions/variables across the script use lower case letters and underscore as a delimiter, let's be consistent with this code style. In your original code you redirect stdin and stdout to /dev/null while sourcing. To be on a safe side let's check if sourcing failed and bail out with meaningful error in case something went wrong.

Reply:
Okay, so logical..

I changed: https://github.com/kt97679/tetris/pull/5/commits/cd1bfb7fcce6d592f9f1262cf63641d0ff3fae96

Reply:
>  Please note that all functions/variables across the script use lower case letters and underscore as a delimiter, let's be consistent with this code style.

So should I change these variable names?

```text
local_LinesCompleted
local_Level
local_Score
local_UseCursorKeys
local_Or
local_Rotate
local_LeftRight
local_Drop
local_Quit
local_ToggleColor
local_ToggleShowNext
local_ToggleThisHelp
local_GameOver
```

Reply:
Thank you very much for the changes!

And yes, it will be great if you will change names of the variables you listed. Thank you :)!